packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -8,53 +8,97 @@ excerpt: >-
+* If you want to control the warning threshold on traffic consumption for an endpoint, you can configure the rate limits settings and customize when your org should trigger a warning event. For instance, if you have CIAM and you've configured a [custom threshold](#warning-notification), a warning alert appears if you reach 90% of the rate limit. If you have Workforce, a warning alert appears if you reach 60% of the rate limit.
 
-The dashboard is available in Self-Service Early Access (EA). You can enable it by going into your org's **Settings** > **Features**. In the **Early Access** section, scroll to the **Rate Limit Investigation Dashboard** setting, and click the toggle button to enable this feature.
+### Open the rate limits report
 
-## Access the dashboard
+In the Admin Console, go to **Reports** > **Rate Limits**. The rate limits report opens in the **Overview** tab.
+
+The **Overview** tab provides you with graphs that plot the number of violations over a three-month period and is based on system log events. It also provides a table that lists your org's APIs and their corresponding current rate limits, violations and warnings. See [Browse and monitor rate limit usage](#browse-and-monitor-rate-limit-usage).
+
+The **Settings** tab allows you to change the per client rate limit settings and customize the threshold for warning notifications when the endpoint's rate limit is exceeded. See [Change rate limit settings](#change-rate-limit-settings).
+
+### Browse and monitor rate limit usage
+
+#### Violation graphs
+
+The Violations line graph plots the number of violations over the selected time period, which can be the last 24 hours, last 7 days, last month, or last 3 months. The Violations per endpoint bar graph shows the top ten number of violations by endpoint over the same time period.
+
+You can hover over the line graph or bar graph to get more details. The details also include a link to the system log where you can view filtered events specific to that endpoint and time period.
+
+#### APIs table
+
+The APIs table shows each endpoint and its corresponding rate limit for your org. The rate limits include all overrides, dynamic scale, or Workforce multipliers. The table also shows the number of violations and warnings for each endpoint in the last hour, and includes a trendline of usage in the past hour.
+
+You can sort on all the columns. The Trendline is sorted by the current usage.
+
+Each endpoint is a link that navigates you to the [dashboard view of endpoint rate limits and violations](#endpoint-rate-limits-and-violations) where you can obtain more details on the endpoint.
+
+You can perform a search or filter your table list by making an entry in the Search textbox. You can also perform a "lookup" with the Search textbox, for example, in cases when you're calling Okta with an example `requestURI` and you want to know the rate limit that will be applied to calls that are made against that `requestURI`.
```

Comment:
The 2nd sentence here is rather long. My mind naturally edited into two shorter sentences lol

Reply:
Fixed.